ci/scripts/gitlab/report_test_results.py

```
@@ -162,18 +165,38 @@ def main():
     args = parser.parse_args()
-    junit_data = parse_junit(args.junit_file)
-    coverage_data = parse_coverage(args.coverage_file)
 
-    report_messages = build_messages(junit_data, coverage_data)
+    return_code = 0
+    try:
+        junit_data = parse_junit(args.junit_file)
+        coverage_data = parse_coverage(args.coverage_file)
+
+        report_messages = build_messages(junit_data, coverage_data)
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)